src/configs/localization/en.json

```diff
@@ -0,0 +1,199 @@
+        "MIN_LENGTH_CHARACTERS": "Field must contain at least {{min}} characters",
+        "EQUAL_OR_LESS": "{{entity}} should be equal to or less than {{equalEntity}}",
+        "invalid-form": "Some form details are incorrect or missing... Please check!",
+        "invalid-order-form": "Some order details are incorrect or missing... Please check!",
+        "MAX_LENGTH_CHARACTERS": "The maximum number of characters for the field is {{max}}"
+      }
+    },
+    "placeholder": {
+      "email": "Email",
+      "roles": "Roles",
+      "prefix": "Prefix",
+      "suffix": "Suffix",
+      "password": "Password",
+      "lastName": "Last Name",
+      "userName": "User Name",
+      "firstName": "First Name",
+      "middleName": "Middle Name",
+      "createdDate": "Creation Date"
+    },
+    "label": {
+      "email": "Email",
+      "roles": "Roles",
+      "prefix": "Prefix",
+      "suffix": "Suffix",
+      "password": "Password",
+      "lastName": "Last Name",
+      "userName": "User Name",
+      "firstName": "First Name",
+      "middleName": "Middle Name",
+      "createdDate": "Creation Date"
+    },
+    "errors": {
+      "details": "Details.",
+      "default": "Something went wrong...",
+      "undefined": "Something went wrong...",
+      "download-file": "File download is not possible.",
+      "crash": "App has crashed. Please reload page and try again.\nIf nothing happens contact contact@abiagenten.com",
+      "ACCESS_DENIED": "Access denied.",
+      "ACCESS_IS_DISABLED": "User is disabled.",
+      "UNKNOWN_ERROR": "Something went wrong...",
+      "SQL_CODE_NULL": "Something went wrong...",
+      "DEAL_DOESNT_EXIST": "Deal doesn't exist.",
+      "EMAIL_NOT_UNIQUE": "Email must be unique.",
+      "TITLE_NOT_UNIQUE": "Title must be unique.",
+      "ORDER_DOESNT_EXIST": "Order doesn't exists.",
+      "TENANT_DOESNT_EXIST": "Tenant doesn't exist.",
+      "RESET_TOKEN_EXPIRED": "Reset token is expired.",
+      "BAD_CREDENTIALS": "Email or password is invalid."
+    }
+  },
+  "WELCOME": {
+    "notifications": {
+      "success": {
+      },
+      "error": {
+      },
+      "confirm": {
+      }
+    },
+    "general": {
+      "title": "WELCOME!",
+      "description": "We are really glad to see you!",
+      "other": "Welcome footer"
+    }
+  },
+  "AUTH": {
+    "notifications": {
+      "success": {
+      },
+      "error": {
+      },
+      "confirm": {
+      }
+    },
+    "actions": {
+      "login": "Login",
+      "forgotPassword": "Forgot Password?"
+    },
+    "general": {
+      "title": "Boilerplate"
+    }
+  },
+  "USERS": {
+    "notifications": {
+      "success": {
+      },
+      "error": {
+      },
+      "confirm": {
+      }
+    },
+    "actions": {
+      "search": "Search",
+      "options": "Options",
+      "remove": "Remove selected",
+      "create": "Create User",
+      "sort": {
+        "name": "Name",
+        "id": "Id",
+        "creationDate": "Creation Date",
```

Comment:
If the field. is dynamic, and the same with entity from backend you can leave it as camelCase